students/itsabhi/Lesson3/slicing.py

```diff
@@ -0,0 +1,47 @@
+def exchange_first_last(a):
+    a_seq = a
+    a_new_seq = a_seq[-1:] + a_seq[1:-1] + a_seq[0:1]
+    return a_new_seq
```

Comment:
you don't need to reassign `a` to `a_seq`, I would rename your input var to `seq` and use it directly there